javascript/javascript1/week4/Voice-Assistant.js

```diff
@@ -0,0 +1,86 @@
+let vname = '';
+const toDo = [];
+function getReply(command) {
+  const arrayOfStrings = command.split(" ");
+  
+  if (command.startsWith("Hello my name is")) {
+    vname = arrayOfStrings[4];
+    return `Nice to meet you ${vname}.` ;
+  }
+  if (command =="What is my name?") {
+    return `Your name is ${vname}.`;
+  }
+  if (command == "Add fishing to my todo") {
+    toDo.push(command);
+    return "fishing added to my todo";
+  }
+  if (command =="Add singing in the shower to my todo") {
+    toDo.push(command);
+    return "singing in the shower";
+  }
+  if (command =="Add dancing to my todo") {
+    toDo.push(command);
+    console.log(toDo)
+    return "dancing added to your todo";
+  }
+  if (command =="remove fishing from my todo") {
```

Comment:
Again same like adding. Make it more dynamic :)

Reply:
done